logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultPathAssertions.kt

```diff
@@ -112,6 +112,15 @@ class DefaultPathAssertions : PathAssertions {
     override fun <T : Path> isDirectory(container: AssertionContainer<T>): Assertion =
         fileTypeAssertion(container, A_DIRECTORY) { it.isDirectory }
 
+    override fun <T : Path> toBeASymbolicLink(container: AssertionContainer<T>): Assertion =
+        changeSubjectToFileAttributes(container, NOFOLLOW_LINKS) { fileAttributesExpect ->
```

Comment:
I see that you copied the code of fileTypeAssertion due to the NOFOLLOW_LINKS.I suggest you add `linkOption: LinkOption? = null` to fileTypeAssertion instead